tpot/base.py

```diff
@@ -771,24 +765,32 @@ def _evaluate_individuals(self, individuals, features, classes, sample_weight=No
 
         # evalurate pipeline
         resulting_score_list = []
-        # chunk size for pbar update
         for chunk_idx in range(0, len(sklearn_pipeline_list), self.n_jobs * 4):
-            jobs = []
-            for sklearn_pipeline in sklearn_pipeline_list[chunk_idx:chunk_idx + self.n_jobs * 4]:
-                job = delayed(_wrapped_cross_val_score)(
-                    sklearn_pipeline,
-                    features,
-                    classes,
-                    self.cv,
-                    self.scoring_function,
-                    sample_weight,
-                    self.max_eval_time_mins
-                )
-                jobs.append(job)
-            parallel = Parallel(n_jobs=self.n_jobs, verbose=0, pre_dispatch='2*n_jobs')
-            tmp_result_score = parallel(jobs)
-            # update pbar
-            for val in tmp_result_score:
+            if self.n_jobs == 1:
+                tmp_scores = [_wrapped_cross_val_score(
+                            sklearn_pipeline,
+                            features,
+                            classes,
+                            self.cv,
+                            self.scoring_function,
+                            sample_weight,
+                            timeout=self.max_eval_time_seconds
+                            ) for sklearn_pipeline in sklearn_pipeline_list[chunk_idx:chunk_idx + self.n_jobs * 4]]
+            else:
+                jobs = []
+                for sklearn_pipeline in sklearn_pipeline_list[chunk_idx:chunk_idx + self.n_jobs * 4]:
+                    job = delayed(_wrapped_cross_val_score)(
+                        sklearn_pipeline,
+                        features,
+                        classes,
+                        self.cv,
+                        self.scoring_function,
+                        sample_weight,
+                        timeout=self.max_eval_time_seconds
+                    )
+                    jobs.append(job)
+                tmp_scores = compute(*jobs, get=multiprocessing.get, num_workers=self.n_jobs)
```

Comment:
There might be some benefit to checking in with Dask's globals before hard-wiring `dask.multiprocessing.get`.  You might try the following instead:

```python
import dask
if njobs == 1:
    get = dask.async.get_sync
elif 'get' in dask.context._globals:
    get = dask.contest._globals['get']
else:
    get = multiprocessing.get

compute(..., get=get)
```

This would allow users to step in with other schedulers, like the distributed scheduler.  Also if you are only using a single core then it's best to avoid the multiprocessing scheduler, which has some non-trivial data movement overhead.